script/benchmarks/send_assessments_job_benchmark.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+# https://ruby-doc.org/stdlib-2.5.0/libdoc/benchmark/rdoc/Benchmark.html
+# https://github.com/evanphx/benchmark-ips
+# https://github.com/SamSaffron/memory_profiler
+# https://github.com/tmm1/stackprof
+
+require_relative '../../config/environment'
+require 'memory_profiler'
+require 'benchmark/ips'
+
+ActionMailer::Base.perform_deliveries = false
+
+timestamp = Time.now.utc.iso8601
+stackprof_file = "script/benchmarks/output/send_assessments_job_benchmark_#{timestamp}_CPU.dump"
+flamegraph_file = "script/benchmarks/output/send_assessments_job_benchmark_#{timestamp}_FLM"
+memprof_file = "script/benchmarks/output/send_assessments_job_benchmark_#{timestamp}_MEM.log"
+benchmark_file = "script/benchmarks/output/send_assessments_job_benchmark_#{timestamp}_BCM.log"
+$stdout = File.new(benchmark_file, 'w')
+$stdout.sync = true
+
+Benchmark.bm(20) do |x|
```

Comment:
@Bialogs I found in my testing that when benchmarking ips and stackprof are used together that stackprof breaks and becomes unusable. Since I figured stackprof is preferred over IPS I am using vanilla benchmarking here

Reply:
That's fine. I think the flamegraph is nice to have. Any thoughts on https://ruby-doc.org/stdlib-2.7.1/libdoc/benchmark/rdoc/Benchmark.html#method-c-bmbm

Reply:
Since this test in particular is expected to take anywhere from 30-60 minutes (or slightly more) then running the test twice with `bmbm` would result in double the testing time.

Reply:
Yeah great point...With the amount of action happening in the `perform_now` it should be fine. Maybe `bmbm` is more for a microbenchmark scenario.